client/package.json

```diff
@@ -13,22 +13,40 @@
     "not ie < 9"
   ],
   "dependencies": {
+    "axios": "^0.19.0",
     "body-parser": "~1.19.0",
+    "bootstrap": "^4.3.1",
```

Comment:
could you show me a place where it is needed? Isn't it wrapped inside `react-bootstrap` package?